src/script/standard.h

```diff
@@ -41,6 +43,11 @@ class CScriptID : public uint160
 };
 
 static const unsigned int MAX_OP_RETURN_RELAY = 8192;      //! bytes
+
+static const unsigned int MAX_OP_RETURN_RELAY_KMD = 140; /* 76 bytes of nota size + 64 bytes chain name */
+static const unsigned int MAX_OP_RETURN_RELAY_AC = 208;  /* 144 bytes of nota size + 64 bytes chain name */
```

Comment:
I think 208 bytes is too low for marmara (although they currently do not sync to komodo)

Reply:
MCL is a different codebase. The values of `MAX_OP_RETURN_RELAY_KMD` and `MAX_OP_RETURN_RELAY_AC` were derived from the maximum possible sizes of notarization OP_RETURN in KMD itself and assetchains. Additionally, before setting these values, all existing chains were scanned to determine the maximum size of useful OP_RETURN, ensuring that everything needed would fit within the new restrictions.